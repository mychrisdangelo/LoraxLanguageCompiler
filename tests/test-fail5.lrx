/* Tests failure of conditional statements */

int main () {
	
	int i;
	i = 1;
	while(i) {
		print(3);
	}
	
}