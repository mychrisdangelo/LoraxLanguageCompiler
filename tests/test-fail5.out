Fatal error: exception Failure("while loop must evaluate on boolean expression")
